cli_tools/common/image/importer/request.go

```diff
@@ -105,6 +105,21 @@ type ImageImportRequest struct {
 	Zone                  string `name:"zone" validate:"required"`
 }
 
+// MergeBYOLIntoOSID interprets the user's --os and --byol flags, and returns a semantically-equivalent
+// tuple. The returned values follow the invariants from ImageImportRequest's validation where --byol may only
+// be specified when --os is empty (implying that detection is being used).
+//
+// For example, given `--byol --os=rhel-8`, this will return `--os=rhel-8-byol`.
+func MergeBYOLIntoOSID(osID string, byol bool) (resolvedOSID string, resolvedBYOL bool) {
```

Comment:
nit: resolvedOSID->mergedOSID, or rename the func to ResolveBYOLIntoOSID to have naming consistent?

Reply:
New version uses "fixed" for method name and to prefix return values